### ISSUE TITLE
Nation NPC's attempting to ram friendlies

### ISSUE THREAD

**Issue by FoxyWoxy7035** [DraconisCluster/DX-Issues#53]
** tl; dr **
Nation NPC's aggressively chase friendly players and has resulted in at least one ramming.

** Full info **
It seems like the NPC's ai is randomly treating some friendlies as hostile players. Their weapons can't actually fire on the friendly grids, but the npc's aggressive flying can result in collisions.

I know of at least three cases, there are likely more. All players reported they had full ownership of their grids. Some of those involved had been in their nation for weeks-months with no issues.

**Screenshots**
Ticket 53177:
![image](https://github.com/user-attachments/assets/334d7f55-694d-43c4-8b92-3081dfdfa674)
From a chase I witnessed, at least 7 npc's chased this friendly small grid all the way to a safezone:
![image](https://github.com/user-attachments/assets/3eb0cf93-84b8-4b5a-826e-1b88f0d17dd9)

**Comment by Terif1** [comment]
Was the fixrep command used and if so was it helpful here? My guess for this one would be that he's not seen as friendly by those npcs and so he is being "attacked" just minus the torps firing (possibly wc not letting a neutral grid be shot?)

**Comment by FoxyWoxy7035** [comment]
We've now run fixrep on all the nations. I will report back if it continues to be an issue. There are at least two open tickets on this who can give us feedback.

**Comment by draconb** [comment]
It's a constant issue. Not sure if it's caused by other people joining their faction or leaving or what is the trigger

**Comment by FoxyWoxy7035** [comment]
Players are still being chased even after we did the fixreps
